custom_components/moonraker/sensor.py

```diff
@@ -178,6 +178,16 @@ class MoonrakerSensorDescription(SensorEntityDescription):
         icon="mdi:timer",
         device_class=SensorDeviceClass.TIMESTAMP,
     ),
+    MoonrakerSensorDescription(
```

Comment:
can you add a test case for it somewhere in this function: https://github.com/marcolivierarsenault/moonraker-home-assistant/blob/258d2958de8d77bc65f9561d14b50c8926ead23f/tests/test_sensor.py#L80